<a href="https://colab.research.google.com/github/Lapitel/llama2-train/blob/main/00_venv_on_colab.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [3]:
# install virtual environment package
!pip install virtualenv

# create virtual environment
!virtualenv myenv



     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.8/3.8 MB 35.1 MB/s eta 0:00:00
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 468.9/468.9 kB 50.2 MB/s eta 0:00:00


Collecting virtualenv
  Downloading virtualenv-20.24.6-py3-none-any.whl (3.8 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.8/3.8 MB 35.1 MB/s eta 0:00:00
Collecting distlib<1,>=0.3.7 (from virtualenv)
  Downloading distlib-0.3.7-py2.py3-none-any.whl (468 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 468.9/468.9 kB 50.2 MB/s eta 0:00:00
Requirement already satisfied: filelock<4,>=3.12.2 in /usr/local/lib/python3.10/dist-packages (from virtualenv) (3.13.1)
Requirement already satisfied: platformdirs<4,>=3.9.1 in /usr/local/lib/python3.10/dist-packages (from virtualenv) (3.11.0)
Installing collected packages: distlib, virtualenv
Successfully installed distlib-0.3.7 virtualenv-20.24.6

In [5]:
!source /content/myenv/bin/activate; pip3 list

Package    Version
---------- -------
pip        23.3.1
setuptools 68.2.2
wheel      0.41.2


Package    Version
---------- -------
pip        23.3.1
setuptools 68.2.2
wheel      0.41.2